lib/ui/src/components/sidebar/__tests__/data.test.ts

```diff
@@ -1,97 +1,69 @@
 import type { StoriesHash } from '@storybook/api';
-import { collapseDocsOnlyStories, collapseAllStories } from '../data';
+import { collapseAllStories } from '../data';
 
 type Item = StoriesHash[keyof StoriesHash];
 
-const docsOnly = { parameters: { docsOnly: true } };
 const root: Item = {
   id: 'root',
   name: 'root',
   depth: 0,
   children: ['a', 'b'],
-  isRoot: true,
-  isComponent: false,
-  isLeaf: false,
+  type: 'root',
 };
 const a: Item = {
   id: 'a',
   name: 'a',
   depth: 1,
-  isComponent: true,
-  isRoot: false,
-  isLeaf: false,
+  type: 'component',
   parent: 'root',
   children: ['a1'],
 };
 const a1: Item = {
   id: 'a1',
   name: 'a1',
-  kind: 'a',
+  title: 'a',
   depth: 2,
-  isLeaf: true,
-  isComponent: false,
-  isRoot: false,
+  type: 'story',
   parent: 'a',
   args: {},
+  prepared: true,
+  importPath: './a.js',
 };
 const b: Item = {
   id: 'b',
   name: 'b',
   depth: 1,
-  isRoot: false,
-  isComponent: true,
-  isLeaf: false,
+  type: 'component',
   parent: 'root',
   children: ['b1', 'b2'],
 };
 const b1: Item = {
   id: 'b1',
   name: 'b1',
-  kind: 'b',
+  title: 'b',
   depth: 2,
-  isLeaf: true,
-  isRoot: false,
-  isComponent: false,
+  type: 'story',
   parent: 'b',
   args: {},
+  prepared: true,
+  importPath: './b1.js',
 };
 const b2: Item = {
   id: 'b2',
   name: 'b2',
-  kind: 'b',
+  title: 'b',
   depth: 2,
-  isLeaf: true,
-  isRoot: false,
-  isComponent: false,
+  type: 'story',
   parent: 'b',
   args: {},
+  prepared: true,
+  importPath: './b2.js',
 };
 
 const stories: StoriesHash = { root, a, a1, b, b1, b2 };
 
-// this seems to work as intended
-describe('collapse docs-only stories', () => {
-  it('ignores normal stories', () => {
-    const filtered = collapseDocsOnlyStories(stories);
-    expect(filtered).toEqual(stories);
-  });
-
-  it('filters out docs-only stories', () => {
-    const hasDocsOnly: StoriesHash = {
-      ...stories,
-      a1: { ...a1, parameters: { ...a1.parameters, ...docsOnly.parameters } },
-    };
-    const filtered = collapseDocsOnlyStories(hasDocsOnly);
-
-    expect(filtered.root.children).not.toContain(hasDocsOnly.a.id);
-    expect(filtered.root.children).toContain(hasDocsOnly.a1.id);
-    expect(filtered.root.children).toContain(hasDocsOnly.b.id);
-
-    expect(filtered.a).not.toBeDefined();
-  });
-});
-
-describe('collapse all stories', () => {
+// FIXME: skipping as we won't need this long term
```

Comment:
Can we just delete this?

Reply:
I'm not dropping it in this PR, so I better get it working